OpenSSL/ASN1.hsc

```diff
@@ -125,26 +130,13 @@ peekASN1Time time
              _ASN1_TIME_print bioPtr time
                   >>= failIf_ (/= 1)
          timeStr <- bioRead bio
-         case parseTimeM True locale "%b %e %H:%M:%S %Y %Z" timeStr of
+#if MIN_VERSION_time(1,5,0)	       
+         case parseTimeM True defaultTimeLocale "%b %e %H:%M:%S %Y %Z" timeStr of
+#else
+         case parseTime defaultTimeLocale "%b %e %H:%M:%S %Y %Z" timeStr of
```

Comment:
`defaultTimeLocale` has exactly the same month names as were specified manually.